### PR TITLE
Renamed service to socket

### PR DESCRIPTION
**Pull request by ab-passos**
Signed-off-by: Andre Passos <andrebritopassos@gmail.com>

### CODE REVIEW

File: plancontext/service.go [hunk 1]
```diff
@@ -10,54 +10,54 @@ import (
 )
 
 var (
-	serviceIDPath = cue.MakePath(
+	socketIDPath = cue.MakePath(
 		cue.Str("$dagger"),
 		cue.Str("service"),
```
Comment (helderco) [inline plancontext/service.go:15]: This can also be changed.

**Review by helderco — APPROVED** [review]
Hey, thanks!

File: plancontext/service.go (hunk 1 above)
Comment (aluzzardi) [reply plancontext/service.go:15]: Hmmm ... can it actually? That would break existing configurations

Comment (helderco) [reply plancontext/service.go:15]: Not on it's own. The cue definition also needs updating. I'll handle this.

**Pull request merged**